src/map.tsx

```diff
@@ -215,10 +237,10 @@ export default class ReactMapboxGl extends Component {
       const didFitBoundsUpdate = (
         fitBounds !== nextProps.fitBounds || // Check for reference equality
         nextProps.fitBounds.length !== (fitBounds && fitBounds.length) || // Added element
-        !!fitBounds.find((c, i) => { // Check for equality
-          const nc = nextProps.fitBounds[i];
-          return c[0] !== nc[0] || c[1] !== nc[1];
-        })
+        !!fitBounds.filter((c, i) => { // Check for equality
```

Comment:
This should be extracted into a helper. Isn't the length here fixed? Because a `filter` on a length of 1 or 2 is pointless